datasets.py

```diff
@@ -984,20 +984,20 @@ def __init__(self, transform):
         # directory storing gps information
         self.gps_folder = '/Users/home/Desktop/research/data/cityscapes/vehicle_trainvaltest/vehicle/train'
 
-        # local boundary shapefile from 
+        # local boundary GeoJSON file from 
         # https://maps.princeton.edu/catalog/stanford-nh891yz3147
-        with open("/Users/home/Downloads/stanford-nh891yz3147-geojson.json") as f:
+        with open("/Users/home/Downloads/stanford-nh891yz3147-GeoJSON.json") as f:
```

Comment:
prob change filepath to be fake Data/stanford....

Reply:
changed